### PR TITLE
adding class global attribute live example

### PR DESCRIPTION
**Pull request by chrisdavidmills**
Part of this work: https://github.com/mdn/sprints/issues/310

### CODE REVIEW

File: live-examples/html-examples/global-attributes/css/class.css [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+}
+
+.output {
+    font: 1rem 'Fira Sans', sans-serif;
+}
+
+.note {
+    font-style: italic;
+    font-weight: bold;
+    Margin: 0 20px;
```
Comment (schalkneethling) [inline live-examples/html-examples/global-attributes/css/class.css]: Whoops, change `Margin` to `margin`

File: live-examples/html-examples/global-attributes/css/class.css [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+}
+
+.editorial {
+    background: rgb(255,0,0,0.25);
```
Comment (schalkneethling) [inline live-examples/html-examples/global-attributes/css/class.css]: Nit: Code style: Space required after commas.

File: live-examples/html-examples/global-attributes/css/class.css [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    padding: 10px;
+}
+
+.editorial:before {
+    content: "Editor: "
```
Comment (schalkneethling) [inline live-examples/html-examples/global-attributes/css/class.css]: Nit: Code style: Use single quotes

**Comment by chrisdavidmills** [comment]
Nits fixed! I ended up just getting rid of the margin line altogether, as I didn't think it was needed.

**Pull request merged**